### PR TITLE
85564 rule update for create avcs uos action

### PR DESCRIPTION
**Pull request by pravinvarade10801**
1. Rule update implementation
2. FT's added

### CODE REVIEW

File: src/UKHO.ERPFacade.Common/IO/Azure/AzureTableReaderWriter.cs [hunk 1]
```diff
@@ -133,7 +134,7 @@ public void ValidateAndUpdateIsNotifiedEntity()
                         ||
                         tableitem.ResponseDateTime.HasValue && ((tableitem.ResponseDateTime.Value - tableitem.RequestDateTime.Value) > TimeSpan.FromMinutes(callBackDuration)))
                     {
-                        _logger.LogWarning(EventIds.WebjobCallbackTimeoutEventFromSAP.ToEventId(), $"Request is timed out for the correlationid : {tableitem.CorrelationId}.");
+                        _logger.LogWarning(EventIds.WebjobCallbackTimeoutEventFromSAP.ToEventId(), "Request is timed out for the correlationid : {tableitem.CorrelationId}.", tableitem.CorrelationId);
```
[the remark this answers is not in the document]
Comment (pravinvarade10801) [reply src/UKHO.ERPFacade.Common/IO/Azure/AzureTableReaderWriter.cs]: > Please change tableitem to tableItem

Done.

File: tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/SAPXmlHelper.cs [hunk 2]
```diff
@@ -676,6 +676,28 @@ private static UoSProductInfo getProductInfo(List<string> products)
...
+            }
+            else
+            {
+                Console.WriteLine(unitOfSalesName + " not found in any Product's inUnitOfSale");
+            }
+
+            return firstProductInfo;
+        }
+
         private static ProductUoSInfo getUoSInfo(string productName)
```
[the remark this answers is not in the document]
Comment (pravinvarade10801) [reply tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/SAPXmlHelper.cs:701]: > 

Regarding getUoSInfo, naming conventions related to FT's will handle in another branch.

**Review by harshal11869 — APPROVED** [review]
Looks Good.

**Review by shrey1501620 — APPROVED** [review]

**Review by JiviteshT — APPROVED** [review]
Looks good

**Pull request merged**